contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "In-depth: PostHog vs Fathom"
+date: 2024-11-11
+author:
+ - bijan-boustani
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_fathom_analytics_e327b58767.jpg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Fathom Analtyics both provide alternatives to Google Analytics. In fact, both explicitly mention it with marketing copy like "Switch to a better Google Analytics alternative."
+
+Both products feature web analytics dashboards with an emphasis on simplicity and privacy. They're ideal choices for customers that want to analyze and understand web traffic but find Google Analytics to be needlessly complicated.
```

Comment:
Feature is kind of a weird word because we use it in so many other contexts. 
```suggestion
Both products have web analytics dashboards with an emphasis on simplicity and privacy. They're ideal choices for customers that want to analyze and understand web traffic but find Google Analytics to be too complicated.
```